schemas/package.yml

```diff
@@ -0,0 +1,88 @@
+---
+- name: package
+  title: Package
+  group: 2
+  short: These fields contain information about an installed software package.
+  description: >
+    These fields contain information about an installed software package. It contains general information about a
+    package, such as name, version or size. It also contains installation details, such as time, URL or
+    user who installed the package.
+  type: group
+  fields:
+
+    - name: id
```

Comment:
How would this ID typically be generated? Is APM already doing this?

Reply:
No APM is not collecting any `package` related data yet. It was mainly inspired by `entity_id` from the [auditbeat system package](https://www.elastic.co/guide/en/beats/auditbeat/current/exported-fields-system.html#_package)

Reply:
Discussion around ID was started in https://github.com/elastic/ecs/pull/532#discussion_r318101822

Reply:
Ah sorry, didn't see that. Let's keep the discussion there.